tests/gui/TestGui.cpp

```diff
@@ -91,11 +95,12 @@ void TestGui::initTestCase()
     // Disable the update check first time alert
     config()->set("UpdateCheckMessageShown", true);
 
-    m_mainWindow.reset(new MainWindow());
-    Bootstrap::restoreMainWindowState(*m_mainWindow);
     Bootstrap::bootstrapApplication();
+
+    m_mainWindow.reset(new MainWindow());
     m_tabWidget = m_mainWindow->findChild<DatabaseTabWidget*>("tabWidget");
     m_mainWindow->show();
+    m_mainWindow->resize(1024, 768);
```

Comment:
🥴

Reply:
There really is no good way around this, we do require a minimum screen size of 1024x768 (not a high bar!). The reason for this is when testing on Hi-DPI screens the window is so small as to prevent clicking some entries in the entry list causing the tests to fail.

Reply:
I know, I know. :shrug: